Add support for Cassandra DB (#5683)
chef/inspec: docs-chef-io/content/inspec/resources/cassandradb_conf.md

+++
title = "cassandradb_conf resource"
draft = false
gh_repo = "inspec"
platform = "os"

[menu]
  [menu.inspec]
    title = "cassandradb_conf"
    identifier = "inspec/resources/os/cassandradb_conf.md cassandradb_conf resource"
    parent = "inspec/resources/os"
+++

> Use the `cassandradb_conf` Chef InSpec audit resource to test the configurations of Cassandra DB, typically located at `$CASSANDRA_HOME/cassandra.yaml` or `$CASSANDRA_HOME\conf\cassandra.yaml` depending upon the platform.

Reviewer: ```suggestion
Use the `cassandradb_conf` Chef InSpec audit resource to test the configuration of a Cassandra database, which is typically located at `$CASSANDRA_HOME/cassandra.yaml` or `$CASSANDRA_HOME\conf\cassandra.yaml` depending upon the platform.
```

## Installation

This resource is distributed along with Chef InSpec itself. You can use it automatically.

## Requirements

> - Value for environment variable `CASSANDRA_HOME` should be set in the system.

Reviewer: ```suggestion
- The value of the `CASSANDRA_HOME` environment variable must be set in the system.
```
